src/Common/test-runtime/project.json

```diff
@@ -8,6 +8,8 @@
     "System.IO.Compression": "4.2.0-beta-devapi-24410-02",
     "System.Runtime.InteropServices.RuntimeInformation": "4.0.1-beta-devapi-24410-02",
     "System.Linq.Parallel": "4.0.2-beta-devapi-24410-02",
+    "System.Private.Xml": "4.0.0-beta-devapi-24410-02",
+    "System.Private.Xml.Linq": "4.0.0-beta-devapi-24410-02",
```

Comment:
Do you mind creating an issue to remove these two once System.Xml.ReaderWriter and System.Xml.XDocument dependencies get fixed?